src/test/groovy/nebula/plugin/dependencylock/tasks/GenerateLockTaskSpec.groovy

```diff
@@ -89,6 +89,44 @@ class GenerateLockTaskSpec extends ProjectSpec {
         task.dependenciesLock.text == lockText
     }
 
+    def 'simple lock for all lockable configurations - without skippedConfigurationsPrefixes'() {
```

Comment:
Can you also add a test using core locking, so that we can expect the same behavior there?

Reply:
This currently doesn't cover core locks as it is more intended for nebula's to avoid locking specific configs. I believe core locks are more on "this is what I want" vs "I will get everything but this..". am .I correct?

Reply:
That is true! 

In a core-lock test with scala, the zinc configurations won't be locked by default, as seen in https://github.com/nebula-plugins/gradle-dependency-lock-plugin/blob/v8.1.0/src/test/groovy/nebula/plugin/dependencylock/DependencyLockPluginWithCoreSpec.groovy#L596-L601

Core locking uses the same logic by calling `lockableConfigurations(...)`, which is what made me think of it. Since the default is `[]`, there is no change to core locking, but it has the option to use this feature. https://github.com/nebula-plugins/gradle-dependency-lock-plugin/blob/v8.1.0/src/main/groovy/nebula/plugin/dependencylock/utils/CoreLockingHelper.groovy#L100

Reply:
oh interesting, I might revisit this then :) or if you want we can chat about it